src/Pos/Block/Logic/Internal.hs

```diff
@@ -111,6 +117,9 @@ applyBlocksUnsafeDo
 applyBlocksUnsafeDo blunds pModifier = do
     -- Note: it's important to put blocks first
     mapM_ putToDB blunds
+    -- If program is interrupted now, we will rollback all wallet sets
```

Comment:
now = when? After this call or before? If I use brain, I think you mean `after` (I may be stupid though and it may be false), but if I just read comment, then I interpret it as `before`.